### ISSUE TITLE
Unable to get SMTP working - are there any logs?

### ISSUE THREAD

**Issue by swilkey** [Jay-Rad/Remotely#78]
Hi Jared,

I installed the remotely server on Ubuntu today (I was initially concerned at what the installation script would do but looked at it and was surprised at how good it was) and it went really well.  Thank you!

I've been testing the server and realised that it is not sending SMTP messages when I try to invite users to access it.  I'm not sure how to debug this as I cannot find any logs being written by remotely server when it tries to send email.  

I tested smtp mail by telnet from the server hosting the remotely web page to my mail server.  I was able to successfully send email from it, unencrypted, over port 25.  Yet, when I have those same settings in the remotely server it is unable to send mail.  Is it trying to send mail encrypted?  Is it doing something else that might mean this test was not a valid test?  Is there a way to see if it is failing or succeeding?

I tried setting the password wrongly on the smtp account in remotely server (which incidentally was not required when sending the message via telnet) and yet the server did not give any error message when I tried to invite a user - exactly the same happened as happens if I use the correct password.

I look forward to your advice.
Stephen

**Comment by Jay-Rad** [comment]
I've added more logging for emails, which will now show up in the Server Logs section in Account.  I've also added error feedback to the Send Invites form, as well as couple others.

I might also add a Test button to the SMTP section in Server Config to test it immediately.

Currently, the email errors are being logged without an organization ID attached to them.  So they can only be viewed by opening the database and looking at the EventLogs table.
